packages/ui/components/combobox/src/LionCombobox.js

```diff
@@ -141,6 +141,64 @@ export class LionCombobox extends LocalizeMixin(OverlayMixin(LionListbox)) {
     ];
   }
 
+  /**
+   * @override ChoiceGroupMixin
+   */
+  // @ts-ignore
+  get modelValue() {
+    const elems = this._getCheckedElements();
+    if (this.multipleChoice) {
+      if (!this.requireOptionMatch) {
+        // eslint-disable-next-line no-console
+        console.warn("multipleChoice and requireOptionMatch can't be used at the same time yet.");
+      }
+      return elems.map(el => el.choiceValue);
+    }
+    if (elems[0]) {
+      return elems[0].choiceValue;
+    }
+    if (!this.requireOptionMatch && this._inputNode) {
+      return this._inputNode.value;
+    }
+    return '';
+  }
+
+  /**
+   * @override ChoiceGroupMixin
+   */
+  set modelValue(value) {
+    /**
+     * @param {ChoiceInputHost} el
+     * @param {any} val
+     */
+    const checkCondition = (el, val) => {
+      if (typeof el.choiceValue === 'object') {
+        return JSON.stringify(el.choiceValue) === JSON.stringify(value);
+      }
+      return el.choiceValue === val;
+    };
+
+    if (this.__isInitialModelValue) {
+      this.registrationComplete.then(() => {
+        this.__isInitialModelValue = false;
+        // @ts-ignore
+        this._setCheckedElements(value, checkCondition);
+        this.requestUpdate('modelValue', this._oldModelValue);
+      });
+    } else {
+      // @ts-ignore
+      this._setCheckedElements(value, checkCondition);
+      this.requestUpdate('modelValue', this._oldModelValue);
+    }
+    this._oldModelValue = this.modelValue;
+  }
+
+  reset() {
+    super.reset();
+    // @ts-ignore
+    this._inputNode.value = this._initialModelValue;
```

Comment:
Here we also should account for multiple choice?

Reply:
Added a test, i don't think anything special is needed here.

Reply:
Yeah, you're right (only in the unlikely edge case where one switches dynamically from .multipleChoice true to false or vice versa).

This is maybe not the best example where multipleChoice should be taken into account, but I would still have a check for .modelValue and IsMatchingOption Validator